src/main/webapp/app/course/learning-paths/components/learning-path-lecture-unit/learning-path-lecture-unit.component.html

```diff
@@ -0,0 +1,25 @@
+@if (loading()) {
+    <div class="row justify-content-center m-4">
+        <div class="spinner-border text-primary" role="status">
+            <span class="sr-only">Loading...</span>
```

Comment:
I don't suppose we could localize this screen reader text, could we?

Reply:
Actually we don't need the `Loading...` here anyway as just a loading spinner will be displayed. I suppose that the `Loading..` is there to show it as an indicator if the loading spinner can not be displayed. (example was copied from bootstrap)